### PR TITLE
polish(jrn): polish mbpo doc

### PR DESCRIPTION
**Pull request by jiaruonan**

### CODE REVIEW

File: source/hands_on/mbpo.rst [hunk 1]
```diff
@@ -73,6 +102,12 @@ P.S.：
 
 1. The above results are obtained by running the same configuration on five different random seeds (0, 1, 2, 3, 4)
```
Comment (PaParaZz1) [inline source/hands_on/mbpo.rst:103]: 3seed

File: source/hands_on/mbpo.rst [hunk 2]
```diff
@@ -3,26 +3,55 @@ MBPO
 
 Overview
 ---------
-Model-based policy optimization (MBPO) was first proposed in `When to Trust Your Model: Model-Based Policy Optimization <https://arxiv.org/abs/1906.08253>`_. MBPO utilizes short model-generated rollouts and provides a guarantee of monotonic improvement at each step. In details, MBPO trains an ensemble of models to fit the transitions of the true environment and uses it to generate short trajectories starting from real environment states to do policy improvement. For the choice of RL policy, MBPO use SAC as its RL component.
+Model-based policy optimization (MBPO) was first proposed in `When to Trust Your Model: Model-Based Policy Optimization <https://arxiv.org/abs/1906.08253>`_. 
+MBPO utilizes short model-generated rollouts and provides a guarantee of monotonic improvement at each step. 
+In details, MBPO trains an ensemble of models to fit the transitions of the true environment and uses it to generate short trajectories starting from real environment states to do policy improvement. 
+For the choice of RL policy, MBPO use SAC as its RL component.
```
Comment (PaParaZz1) [inline source/hands_on/mbpo.rst:9]: add a link to `awesome` repo

File: source/hands_on/mbpo.rst [hunk 3]
```diff
@@ -3,26 +3,55 @@ MBPO
...
+
+.. math::
+  Q^\pi(\boldsymbol{s}_t,\boldsymbol{a}_t) = \mathbb{E}_\pi[{\sum}_{t=0}^{\infty}\gamma^t r(\boldsymbol{s}_t,\boldsymbol{a}_t)]
+
+Policy improvement step:
+
+.. math::
+  J_\pi(\phi, D) = \mathbb{E}_{s_t \sim D}[D_{KL}(\pi \| exp\{Q^\pi - V^\pi\})]
+
+
```
Comment (PaParaZz1) [inline source/hands_on/mbpo.rst:54]: talk something about `a guarantee of monotonic improvement`

**Review by PaParaZz1 — CHANGES_REQUESTED** [review]

File: source/hands_on/mbpo.rst (hunk 1 above)
Comment (jiaruonan) [reply source/hands_on/mbpo.rst:103]: All done.

**Review by PaParaZz1 — APPROVED** [review]

**Pull request merged**